process/sync/baseSync.go

```diff
@@ -135,13 +139,10 @@ func (boot *baseBootstrap) applyBlock(
 		return err
 	}
 
-	headerNoncePool := boot.headersNonces
-	if headerNoncePool == nil {
-		return process.ErrNilDataPoolHolder
+	if header.GetRound() > boot.boostrapRoundIndex {
```

Comment:
the previous for does not know about the bootstrapRoundIndex. Is it blockFinality ? If yes, then rename the variable.

Reply:
bootstrapRoundIndex is a global variable of baseBoostrap structure and it is known in whole class. It is passed from main initialization and it's came from the given flag

Reply:
It represents the round index until which the node will bootstrap from storage. After this it will request the needed info from network and it will also delete all the info with a greater round than this parameter which is stored in its data base